tests/compas/geometry/test_polygon.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+
+from compas.geometry import Point
+from compas.geometry import Polygon
+from compas.utilities import pairwise
+
+
+def test_polygon():
+    points = [[0, 0, x] for x in range(5)]
+    polygon = Polygon(points)
+    assert polygon.points == points
+    assert polygon.lines == [(a, b) for a, b in pairwise(points + points[:1])]
+
+
+def test_equality():
+    points1 = [[0, 0, x] for x in range(5)]
+    polygon1 = Polygon(points1)
+    points2 = [[0, 0, x] for x in range(6)]
+    polygon2 = Polygon(points2)
+    points3 = [[0, 0, x] for x in range(5)] + [[0, 0, 0]]
+    polygon3 = Polygon(points3)
+    assert polygon1 == polygon1
+    assert polygon1 == points1
+    assert points1 == polygon1
+    assert polygon1 != polygon2
+    assert polygon2 != polygon1
```

Comment:
do the `__ne__` still have to be added for `ipy` as well?

Reply:
It was added in `Primitive`, so it is inherited. or do you mean something else?

Reply:
no. didn't think of that...